### DISCUSSION TITLE
How can we create a form in a feature comparison format?

### DISCUSSION THREAD

**Discussion by a1iraxa** [filamentphp/filament#11397] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Is there a way to create a form similar to the "feature comparison format" where on the left side, there will be static labels for the input text fields, and on the right side, there will be multiple columns for the input text fields?

<img width="782" alt="Estimate-Form" src="https://github.com/filamentphp/filament/assets/15581553/3a05786b-0125-4342-8a4f-ebc618cc548b">

I have a repeater field, and each column will represent one item from the repeater, and in each item, I have multiple text fields.

**Comment by leandrocfe** [comment]
Maybe using the table repeater
https://filamentphp.com/plugins/awcodes-table-repeater
